src/main/resources/digital/slovensko/autogram/ui/gui/settings-dialog.fxml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<BorderPane minWidth="800" prefWidth="800" xmlns="http://javafx.com/javafx/17.0.2-ea"
+            xmlns:fx="http://javafx.com/fxml/1">
+    <center>
+        <VBox fx:id="mainBox" maxHeight="600" prefWidth="800.0">
+            <VBox>
+                <TabPane maxHeight="600" prefWidth="800.0"
+                         xmlns="http://javafx.com/javafx" xmlns:fx="http://javafx.com/fxml">
+                    <tabs>
+                        <Tab styleClass="" text="Podpisovanie">
+                            <content>
+                                <AnchorPane styleClass="autogram-tab-header,borders">
+                                    <VBox maxWidth="725" minWidth="725"
+                                          styleClass="autogram-prose" style="-fx-padding: 20px;">
```

Comment:
Tymto custom style sa treba vyhnut, nebudeme to vediet prestylovat v css theme inej.